uiowa/uiowa
# Admissions kiosk work

Closes #3841 


# How to test

- `composer install`
- `blt ds --site=admissions.uiowa.edu`
- `drush @admissions.local cim`
- `yarn workspace admissions_core build`
- `drush @admissions.local cr`
- Go to https://admissions.local.drupal.uiowa.edu/kiosk as anonymous user in Chrome and verify that you can print multiple PDFs
- Verify that page refreshes and resets checkboxes after closing print dialog
- Verify that print button only appears if aos checkboxes are checked
- Verify reset button appears when applying a filter (switch on academic interest)
- Verify that multiple page pdfs are clearly separated on separate pages
- Verify page is set to noindex in metatag settings
